set.mm

```diff
@@ -552990,12 +552990,1267 @@ A collection of theorems for commuting equalities (or
       ( cv cvv wcel el3v3 el2v2 ) ABCACFGHBDEIJ $.
   $}
 
+  ${
+    bian.1 $e |- ( ph <-> ( ch /\ ps ) ) $.
+    $( Commuting conjunction in a biconditional.  (Contributed by Peter Mazsa,
+       17-Jun-2018.) $)
+    bian $p |- ( ph <-> ( ps /\ ch ) ) $=
+      ( wa ancom bitr4i ) ACBEBCEDBCFG $.
+  $}
+
+  ${
+    biand.1 $e |- ( ph -> ( ps <-> ( th /\ ch ) ) ) $.
+    $( Commuting conjunction in a biconditional, deduction form.  (Contributed
+       by Peter Mazsa, 3-Oct-2018.) $)
+    biand $p |- ( ph -> ( ps <-> ( ch /\ th ) ) ) $=
+      ( wa ancom syl6bb ) ABDCFCDFEDCGH $.
+  $}
+
```

Comment:
Wouldn't ~biancom and ~ biancomd better names?

Reply:
@avekens: ~biancom and ~ biancomd are much better. I'll change them everywhere when all open questions are closed below.